### ISSUE TITLE
Simpler notifications for puzzles on demand

### ISSUE THREAD

**Issue by jpd236** [jpd236/Big-Board#9]
Add ability to sign up easily for progress or blocker updates on specific puzzles without having to follow the Slack channel or open the sheet.

**Comment by jpd236** [comment]
Technically this is probably feasible. Registering would be a matter of tracking a set of members who have opted into puzzle status updates, e.g. either via Big Board or the Slack bot. Then, whenever a puzzle's status changes, we DM registered users in Slack with the update.

In practice, I'm not sure how often people really update the blocker status, particularly from blocked -> unblocked. I think it typically sits in blocked until it gets actually solved. Maybe the solve notification is still useful.

I'll take a quick look to see if this can be done without too much trouble for this year.

**Comment by jpd236** [comment]
Done, at least as I interpreted the request.

Each puzzle page in Big Board now has a "Get a Slack DM whenever this puzzle's status changes" checkbox. If you select this, you'll get a Slack DM any time the status changes (e.g. stuck, priority, low priority) or if the puzzle is solved or the solution is updated/cleared.
